myblog/settings.py

```diff
@@ -40,6 +41,7 @@
     "django.contrib.sites",
     "allauth",
     "allauth.account",
+    "allauth.socialaccount.providers.github",
```

Comment:
You're missing this line:

    "allauth.socialaccount",

Right before line 44.

After adding it, run the following:

```
python manage.py makemigrations
python manage.py migrate
python manage.py runserver
```

Try to navigate to: http://127.0.0.1:8000/accounts/github/login/

It should work now. I have to say, this one gave me a good run for my money, it was difficult to find :-)